packages/scheduler/src/__tests__/SchedulerProfiling-test.js

```diff
@@ -498,7 +498,7 @@ Task 1 [Normal]              │                    █████████
       taskId++;
       const task = scheduleCallback(NormalPriority, () => {});
       cancelCallback(task);
-      await waitForAll([]);
+      Scheduler.unstable_flushAll();
```

Comment:
Making this async (which I did during one of the recent codemods) caused the test to time out sometimes. Not consistently, though, which is why I didn't catch it before. Since this is a test of Scheduler itself it's fine to use the lower level, synchronous flushing method.